internal/manifest/list.go

```diff
@@ -93,6 +94,7 @@ type ListEdit struct {
 	UpdateDigest                digest.Digest
 	UpdateSize                  int64
 	UpdateMediaType             string
+	UpdateArtifactType          string
```

Comment:
We don’t really _need_ this feature right now. And for (1), we would also need to add an `UpdateAffectArtifactType` (or rename existing `UpdateAffectAnnotations` to `UpdateInternalCaller` to cover both fields). It seems simpler to me to just not add this field, for now. (Or do you have short-term plans to use it?)

Reply:
No short-term plans, will drop this field.